test/data/utilsData.ts

```diff
@@ -0,0 +1,5 @@
+const data = {
```

Comment:
Recommend to have `fixtures` folder instead of `data`.

Reply:
rename the folder to fixtures?

Reply:
Yes